docs/extras/project-ideas/index.md

```diff
@@ -0,0 +1,176 @@
+
+Building such a body of knowledge, this could then be triangulated with the rest of the OpenTrials database, identifying new academic publications related to trials, for example.
+
+#### Skills
+
+- Advanced data wrangling
+- Intermediate Python
+
+### Collect new data sources
+
+#### What
+
+In OpenTrials, we have [collectors](https://github.com/opentrials/collectors), which use modules and functions of Python code to get data from some external source into the OpenTrials warehouse database for further processing.
+
+There are many data sources available. So far we've build out the spine of the database with core clinical trial records, but there are tens of additional sources that can potentially be matched onto the clinical trials we know about, and provide value to the database.
+
+#### How
+
+See the [documentation for all existing collectors](https://github.com/opentrials/collectors/tree/master/docs/collectors) and start your own with a new data source! A good place to begin would be to see if any collectors have not yet been written for the 16 [WHO Primary Registries](http://www.who.int/ictrp/network/primary/en/).
+
+#### Skills
+
+- Intermediate Python
+- Some basic domain knowledge (to identify additional sources to collect and persist in the OpenTrials warehouse)
+
+### Probabilistic record matching for trial data
+
+#### What
+
+The OpenTrials database threads trial information together from a range of disparate sources. This is currently done via exact matching using unique trial identifiers. Due to the lack of universal standards for identifiers, the presence of multiple identifier types, and the fact that not all records even have an identifier, it is not possible to make all links with exact matching.
+
+Probabilistic matching will allow us to build up a knowledge base of likely matches across trials based on similarities on key data fields.
+
+#### How
+
+Write one or many [processors](https://github.com/opentrials/processors) that do matching across all trial records using probabilistic techniques.
+
+#### Skills
+
+- Advanced data science/wrangling
+- Intermediate Python
+
+### Basic outlier detection
+
+#### What
+
+There is much that can potentially be learned just by looking for outliers in a large dataset. Even without specific domain knowledge, building a set of queries to find outliers can be a first step in asking questions about the data. As a starting point, consider such possibilities around participation, location, age range or interventions for a particular condition.
+
+#### How
+
+Using the [OpenTrials query interface](https://query.opentrials.net/), build a set of queries designed for outlier detection across key data fields.
```

Comment:
I don't think we should give people access to our production re:dash instance for now. At least not before we have a read-only replica of our database, or some other mechanism to make sure slow queries don't hit us too hard.